app/src/ui/clone-repository/clone-repository.tsx

```diff
@@ -616,6 +622,14 @@ export class CloneRepository extends React.Component<
     return url
   }
 
+  private cloneIfCloningEnabled = () => {
```

Comment:
If we update the type signature here to accept a `ClickSource` we can move the check, `if (source.kind === 'keyboard' && source.event.key === 'Enter')` from the child component.

Reply:
We could then rename this to function `onRowKeyDown`.

Reply:
Both changes sound great! Thanks again for the feedback.
I did not like the name `cloneIfCloningEnabled` much at all, so I'm more than happy to make that change ASAP. :+1: